### PR TITLE
feat: adding new features and fixing labelling

### PR DESCRIPTION
**Pull request by YazeedLoonat**
- Addresses [#issue](https://github.com/CityOfDetroit/bloom/issues/1290)

A follow up script will be put out to make these changes to the live site's data, but these changes need to happen in live before that can happen

## Description
This adds the new features and changes the labels for a few existing labels, per the spreadsheet: 
https://docs.google.com/spreadsheets/d/15KIX_wSuKmrtjYYrgRgb-LN1cdxIIJ9RSffPJ7SQ7j4/edit#gid=2017399755

## How Can This Be Tested/Reviewed?
**on the partner site**
log in and go to edit/create a listing
notice the new features are present on the listing form, and notice that the updated label presents are there 
notice that after the save the detail page has the same updates

**on the public site**
take a look at the listing detail page, and notice that the new/updated features are present
take a look at the listing filter form, and notice that the new/updated features are present
notice that the new features do function in the form

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [x] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] I have exported any new pieces added to ui-components
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [x] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: ui-components/src/locales/general.json [hunk 1]
```diff
@@ -1523,6 +1523,19 @@
   "eligibility.accessibility.title": "Accessibility Features",
   "eligibility.accessibility.visual": "Visual",
   "eligibility.accessibility.wheelchairRamp": "Wheelchair Ramp",
+  "eligibility.accessibility.inUnitWasherDryer2": "In-unit washer/dryer",
```
Comment (emilyjablonski) [inline ui-components/src/locales/general.json:1526]: We should alphabetize these, and then I'm curious as to why we have some strings with the `2` label? If we aren't using the old ones, can we just remove them?

**Review by emilyjablonski — CHANGES_REQUESTED** [review]
We touched based offline about some refactoring of the new strings 👍

**Comment by YazeedLoonat** [comment]
Hey @emilyjablonski got this updated per our convo if you have a chance to take another look please

**Comment by emilyjablonski** [comment]
@YazeedLoonat Just want to confirm that Kathy's response was to not google translate and add the new strings in?

File: backend/core/src/migration/1654190736229-new-accessibility-features.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+export class newAccessibilityFeatures1654190736229 implements MigrationInterface {
+    name = 'newAccessibilityFeatures1654190736229'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "listing_features" ADD "barrier_free_unit_entrance" boolean`);
+        await queryRunner.query(`ALTER TABLE "listing_features" ADD "lowered_light_switch" boolean`);
+        await queryRunner.query(`ALTER TABLE "listing_features" ADD "barrier_free_bathroom" boolean`);
+        await queryRunner.query(`ALTER TABLE "listing_features" ADD "wide_doorways" boolean`);
+        await queryRunner.query(`ALTER TABLE "listing_features" ADD "lowered_cabinets" boolean`);
+        await queryRunner.query(`ALTER TABLE "listings" DROP COLUMN "marketing_season"`);
```
Comment (emilyjablonski) [inline backend/core/src/migration/1654190736229-new-accessibility-features.ts:12]: Don't think we want to drop marketing season?

Comment (YazeedLoonat) [reply backend/core/src/migration/1654190736229-new-accessibility-features.ts]: good catch, not sure why the auto gen for migrations pulled that in, got it removed!

**Comment by YazeedLoonat** [comment]
Hey @emilyjablonski I asked kathy and she had me run the translations myself, so those should be in place now

lemme know if you catch anything else!

**Review by emilyjablonski — APPROVED** [review]
LGTM ty for the translations!

**Pull request merged**